src/celery.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 nohup celery -A geonode.celery_app:app beat -l DEBUG -f /var/log/celery.log &>/dev/null &
-nohup celery -A geonode.celery_app:app worker --without-gossip --without-mingle -Ofair -B -E -s django_celery_beat.schedulers:DatabaseScheduler --loglevel=INFO --concurrency=2 -n worker1@%h -f /var/log/celery.log &>/dev/null &
+nohup celery -A geonode.celery_app:app worker --without-gossip --without-mingle -Ofair -B -E --statedb=worker.state --scheduler=celery.beat:PersistentScheduler --loglevel=INFO --concurrency=2 -n worker1@%h -f /var/log/celery.log &>/dev/null &
```

Comment:
Here we are not considering the `CELERY_WORKER_CONCURRENCY `env var, should we replace it even here?

Reply:
Not sure, again I just did a copy from the current geonode-project 3.3.x template so maybe we should first make this change upstream?

Reply:
@mattiagiupponi that file is not the one used by docker, it is just a sample .sh file. `celery-cmd` takes the concurrency variable into account already.